.editorconfig

```diff
@@ -9,8 +9,12 @@ indent_style = space
 indent_size = 4
 insert_final_newline = true
 trim_trailing_whitespace = true
+eclint_block_comment_start = /*
+eclint_block_comment = *
+eclint_block_comment_end = */
 
 [*.md]
+indent_size = 2
```

Comment:
🤔 why?

Reply:
sorry, missed those changes at first.

Reply:
Why did you resolve this conversation? The comment was just explanation that I missed those two things in my first review. Why 2 indents in .md files?

Reply:
Because Markdown files in this repository are indented with 2 spaces.
This fact made me add this line.
Try downloading the single binary from https://github.com/greut/eclint
Starting simply as `eclint` will find everything okay.
IF you revert this line there will be errors.

Reply:
🤔 I haven't downloaded it yet...but I guess that must be about those indents where I just added line breaks in listings and then you have to add the two spaces, so it's rendere correctly as belonging to the same list point. Like here:
```md
- Static method to create PSR-7 `Uri` object
  (`Url::parsePsr7('https://...')`).
``` 

But the actual indentation for e.g. sub list points is 4, like here:
```md
- Support for PHP 8.0
    - Minor change in Vali
```

Reply:
I also use 4 space indent everywhere 🙃 even in *.* files.
https://github.com/szepeviktor/byte-level-care/blob/master/README.md?plain=1

markdownlint keeps an eye on those indents.
https://github.com/szepeviktor/byte-level-care/blob/master/.markdownlint-cli2.yaml

Reply:
OK...I don't know...my editorconfig Plugin has no Problem with 
```md
- Static method to create PSR-7 `Uri` object
  (`Url::parsePsr7('https://...')`).
```

but having indent 4 configured 🤷🏻‍♂️ I'd say let's remove that line, please.

Reply:
Done.
All my energy is gone.
Please continue without me.